src/actions/makeArchive.ts

```diff
@@ -0,0 +1,40 @@
+import { withActionOutput } from '../utils';
+
+import WFArchiveFormat from '../interfaces/WF/WFArchiveFormat';
+import WFSerialization from '../interfaces/WF/WFSerialization';
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Make Archive Action. Makes an archive out of the files passed as input.
```

Comment:
Could you add the rest of the description to this comment please? It should read:

```
 * Make Archive Action. Makes an archive out of the files passed as input. Supports creating zip, tar.gz, tar.bz2, tar, gzip, cpio or iso archives.
```

Thanks!